### PR TITLE
update FAQ - where to get tested.

### PR DESCRIPTION
**Pull request by dsarkar**

### CODE REVIEW

File: src/data/faq.json [hunk 1]
```diff
@@ -65,14 +65,10 @@
...
-                            "</li><li>Appointments only for their own employees:",
-                            "<ul><li>The Federal Chancellery tests its own employees</li><li>Saarland's Ministry of Education and Culture tests teachers</li></ul>",
-                            "</li></ul><hr>An overview of the <b>PCR and rapid antigen test sites</b> can be found on this <a href='https://www.zusammengegencorona.de/testen/?articlefilter=alleartikel' target='_blank' rel='noopener noreferrer'>BMG webpage</a> (not clear whether the test site is connected to the CWA or not).",
+                            "Meanwhile, more than 100 rapid test partners are connected to the CWA infrastructure and can provide their test results to the app.",
+                            "A local search that shows the compatible partners with nearby testing sites in your area can be found here: <a href='https://map.schnelltestportal.de' target='_blank' rel='noopener noreferrer'>map.schnelltestportal.de</a>",
+                            "The list of test sites listed contains only those entries that have been reported by the partners concerned.",
```
Comment (MikeMcC399) [inline src/data/faq.json:70]:
```suggestion
                            "The list of test sites contains only those entries that have been reported by the partners concerned.",

```

Comment (Ein-Tim) [reply src/data/faq.json:70]: Empty line!

File: src/data/faq.json [hunk 2]
```diff
@@ -65,14 +65,10 @@
...
+                            "<hr>An overview of the <b>PCR and rapid antigen test sites</b> can be found on this <a href='https://www.zusammengegencorona.de/testen/?articlefilter=alleartikel' target='_blank' rel='noopener noreferrer'>BMG webpage</a> (not clear whether the test site is connected to the CWA or not).",
                             "<a href='https://www.zusammengegencorona.de/testen/?articlefilter=alleartikel' target='_blank' rel='noopener'><img src='../../assets/img/faq/BMG_testzentren.png' alt='image' width='300'></a>",
                             "Another list of test sites from the Github community: <a href='https://github.com/corona-warn-app/cwa-documentation/issues/569' target='_blank' rel='noopener noreferrer'>Where to get tested?</a>"
```
Comment (MikeMcC399) [inline src/data/faq.json:73]:
```suggestion
                            "Another list of test sites from the GitHub community: <a href='https://github.com/corona-warn-app/cwa-documentation/issues/569' target='_blank' rel='noopener noreferrer'>Where to get tested?</a>"

```

File: src/data/faq_de.json [hunk 3]
```diff
@@ -66,14 +66,10 @@
...
-                            "</li><li>Ohne zentrale Website zur Terminbuchung:",
-                            "<ul><li>Doctorbox</li><li>Huber Health Care</li></ul>",
-                            "</li><li>Termine nur für die eigenen Mitarbeiter",
-                            "<ul><li>Das Bundeskanzleramt testet eigene Mitarbeiter*innen</li><li>Das Ministerium für Bildung und Kultur Saarland testet Lehrer*innen</li></ul>",
-                            "</li></ul><hr>Eine Übersicht der <b>PCR- und Schnell-Teststellen</b> finden Sie auf dieser <a href='https://www.zusammengegencorona.de/testen/?articlefilter=alleartikel' target='_blank' rel='noopener noreferrer'>BMG Seite</a> (nicht ersichtlich ob die Teststelle an die CWA angebunden ist oder nicht).",
+                            "Mittlerweile wurde mehr als 100 Schnelltest-Partner an die CWA-Infrastruktur angeschlossen und können die Ergebnisse in die App übermitteln.",
```
Comment (MikeMcC399) [inline src/data/faq_de.json:69]:
```suggestion
                            "Mittlerweile wurden mehr als 100 Schnelltest-Partner an die CWA-Infrastruktur angeschlossen und können die Ergebnisse in die App übermitteln.",

```

File: src/data/faq_de.json [hunk 4]
```diff
@@ -66,14 +66,10 @@
...
+                            "Eine lokale Suche, die Ihnen in der Umgebung die kompatiblen Partner mit den Teststellen anzeigt, finden Sie hier: <a href='https://map.schnelltestportal.de' target='_blank' rel='noopener noreferrer'>map.schnelltestportal.de</a>",
+                            "Die Liste der aufgeführten Teststellen enthält nur die Einträge, die von den betreffenden Partnern gemeldet wurden.",
+                            "<hr>Eine Übersicht der <b>PCR- und Schnell-Teststellen</b> finden Sie auf dieser <a href='https://www.zusammengegencorona.de/testen/?articlefilter=alleartikel' target='_blank' rel='noopener noreferrer'>BMG Seite</a> (nicht ersichtlich ob die Teststelle an die CWA angebunden ist oder nicht).",
```
Comment (MikeMcC399) [inline src/data/faq_de.json:72]:
```suggestion
                            "<hr>Eine Übersicht der <b>PCR- und Schnell-Teststellen</b> finden Sie auf dieser <a href='https://www.zusammengegencorona.de/testen/?articlefilter=alleartikel' target='_blank' rel='noopener noreferrer'>BMG Seite</a> (nicht ersichtlich, ob die Teststelle an die CWA angebunden ist oder nicht).",

```

File: src/data/faq_de.json [hunk 5]
```diff
@@ -66,14 +66,10 @@
...
                             "<a href='https://www.zusammengegencorona.de/testen/?articlefilter=alleartikel' target='_blank' rel='noopener'><img src='../../assets/img/faq/BMG_testzentren.png' alt='image' width='300'></a>",
                             "Weitere Liste mit Teststellen aus der Github-Community : <a href='https://github.com/corona-warn-app/cwa-documentation/issues/569' target='_blank' rel='noopener noreferrer'>Wo kann man sich testen lassen?</a>"
```
Comment (MikeMcC399) [inline src/data/faq_de.json:74]:
```suggestion
                            "Weitere Liste mit Teststellen aus der GitHub-Community : <a href='https://github.com/corona-warn-app/cwa-documentation/issues/569' target='_blank' rel='noopener noreferrer'>Wo kann man sich testen lassen?</a>"

```

**Review by MikeMcC399 — CHANGES_REQUESTED** [review]
Just some minor suggestions added.

**Comment by Ein-Tim** [comment]
@MikeMcC399 Your suggestions will add empty lines. Please remove them (this is not your fault, GitHubs system is a bit strange here).

**Comment by MikeMcC399** [comment]
@Ein-Tim 
> Your suggestions will add empty lines. Please remove them (this is not your fault, GitHubs system is a bit strange here).

Thanks for your hint! I hadn't noticed and I only recently started using this method after I saw that you were using it.

I have removed all the extra lines added by GitHub!

**Comment by dsarkar** [comment]
@MikeMcC399 and @Ein-Tim Thanks! Great help!

**Pull request merged**